.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "moby",
+  "build": {
+    "context": "..",
+    "dockerfile": "../Dockerfile",
+    "target": "dev"
+  },
+  "workspaceFolder": "/go/src/github.com/moby/moby",
+  "workspaceMount": "source=${localWorkspaceFolder},target=/go/src/github.com/moby/moby,type=bind,consistency=cached",
```

Comment:
Might want to use docker/docker for now (until we renamed the code)

Reply:
Yeah, need to change this, the IDE goes crazy since it doesn't know that there is a redirect